### PR TITLE
POC php cs fixer config

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -45,6 +45,12 @@
     'nullable_type_declaration_for_default_null_value' => ['use_nullable_type_declaration' => true],
     'ordered_class_elements' => true,
     'ordered_imports' => ['sort_algorithm' => 'alpha', 'imports_order' => ['class', 'function', 'const']],
+    'phpdoc_order' => ['order' => ['param', 'throws', 'return', 'phpstan-param', 'psalm-param', 'phpstan-return', 'psalm-return']],
+    'phpdoc_separation' => ['groups' => [
+        ['var', 'phpstan-var', 'psalm-var'],
```
Comment (VincentLanglet) [inline .php-cs-fixer.dist.php:50]: Do we prefer
```
/**
 * @var Foo
 * @phpstan-var Foo<T>
 */
```
or
```
/**
 * @var Foo
 * 
 * @phpstan-var Foo<T>
 */
```

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -45,6 +45,12 @@
...
+        ['phpstan-template', 'phpstan-extends', 'phpstan-implements', 'phpstan-param', 'psalm-param', 'phpstan-return', 'psalm-return'],
```
Comment (VincentLanglet) [inline .php-cs-fixer.dist.php:51]: Do we prefer
```
/**
 * @param Foo
 *
 * @return Foo
 * 
 * @template T
 * @phpstan-param Foo<T>
 * @phpstan-return Foo<T>
 */
```
or
```
/**
 * @param Foo
 *
 * @return Foo
 * 
 * @template T
 * 
 * @phpstan-param Foo<T>
 * 
 * @phpstan-return Foo<T>
 */
```

File: .php-cs-fixer.dist.php [hunk 3]
```diff
@@ -45,6 +45,12 @@
...
+        ['psalm-suppress', 'phpstan-ignore-next-line'],
```
Comment (VincentLanglet) [inline .php-cs-fixer.dist.php:52]:
```
/**
 * @psalm-suppress Foo
 * @phpstan-ignore-next-line
 */
```
or 
```
/**
 * @psalm-suppress Foo
 * 
 * @phpstan-ignore-next-line
 */
```

**Comment by VincentLanglet** [comment]
I'll wait for your review before a dev-kit pr @jordisala1991

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (jordisala1991) [reply .php-cs-fixer.dist.php:50]: I prefer to separate all the phpstan- or psalm- and group them together. Wdyt?

File: .php-cs-fixer.dist.php (hunk 2 above)
Comment (jordisala1991) [reply .php-cs-fixer.dist.php:51]: I prefer first one, you?

File: .php-cs-fixer.dist.php (hunk 3 above)
Comment (jordisala1991) [reply .php-cs-fixer.dist.php:52]: I prefer first one, you?

File: .php-cs-fixer.dist.php (hunk 2 above)
Comment (VincentLanglet) [reply .php-cs-fixer.dist.php:51]: me too

File: .php-cs-fixer.dist.php (hunk 3 above)
Comment (VincentLanglet) [reply .php-cs-fixer.dist.php:52]: me too

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (VincentLanglet) [reply .php-cs-fixer.dist.php:50]: We generally separate phpstan tag, but seems like for @phpstan-var we didnt.
But I can update the config.

Comment (jordisala1991) [reply .php-cs-fixer.dist.php:50]: It seems more consistent to me.

Comment (VincentLanglet) [reply .php-cs-fixer.dist.php:50]: I opened https://github.com/sonata-project/dev-kit/pull/2166 then

**Pull request closed without merging**